tools/test.py

```diff
@@ -255,6 +256,9 @@ def HasRun(self, output):
         logger.info('#' + l)
       for l in output.output.stdout.splitlines():
         logger.info('#' + l)
+    elif output.HasSkipped():
+      skip = is_skipped.findall(output.output.stdout)      
```

Comment:
@bnoordhuis the reason i went with `HasSkipped` was just to follow conventions in test.py as close as possible.